src/controllers/user.js

```diff
@@ -65,3 +65,30 @@ export const updateById = async (req, res) => {
 
   return sendDataResponse(res, 201, { user: { cohort_id: cohortId } })
 }
+
+// NEW //
+export const updateProfile = async (req, res) => {
+  const newUserProfile = await User.fromJson(req.body)
+  const userToUpdateId = Number(req.params.id)
+  newUserProfile.id = userToUpdateId
+
+  console.log('newUserProfile', newUserProfile)
+
+  try {
+    const existingUser = await User.findById(userToUpdateId)
+    console.log('do you exist?', existingUser)
```

Comment:
We can remove these console logs now. The only time we want to keep a console.log is in a catch block so we can see what error has happened.